src/timesync/timesyncd-manager.h

```diff
@@ -3,6 +3,11 @@
 
 #include <sys/timex.h>
 
+#include <openssl/bio.h>
```

Comment:
there are \#if HAVE_OPENSSL in timesyncd-manager.c so shouldn't these includes be wrapped too?